### PR TITLE
feat: game in progress and settings

### PR DESCRIPTION
**Pull request by cindyhalim**
**changes:**
- remove db call from getSerializedCurrentGame
- send game ended event to all pool players
- display game in progress if player joins room with current game
- settings screen for admin -> ability to end game early

https://user-images.githubusercontent.com/52426595/163695132-64a5a944-c60e-4953-9b94-282df5ae16c3.mp4

updated score board title logic:
if game ended before players earn points:
Screen Shot 2022-04-16 at 9 00 20 PM

else:
Screen Shot 2022-04-16 at 9 03 21 PM

### CODE REVIEW

File: backend/src/functions/onCardDrawn.ts [hunk 1]
```diff
@@ -29,23 +31,23 @@ export const handler = async (event: APIGatewayEvent) => {
...
     const nextPlayerId = game.players[(currentPlayerIndex + 1) % game.players.length].id
 
-    const updatedGame: IGameModel = {
+    updatedGame = {
       ...game,
       currentPlayerId: nextPlayerId,
       players: updatedGamePlayers,
     }
     console.log("updating game in db")
     await database.room.updateGame({ roomId, game: updatedGame })
```
Comment (emilyhfdong) [inline backend/src/functions/onCardDrawn.ts]: super nit and unrelated to your work but we can move line 39/40 and 46/47 out of the if block now

File: frontend/src/components/button.tsx [hunk 2]
```diff
@@ -48,7 +48,6 @@ export const Button: React.FC<IButtonProps> = (props) => {
     <RButton
       {...props}
       sx={{
-        zIndex: 100,
```
Comment (emilyhfdong) [inline frontend/src/components/button.tsx:51]: hehe

File: frontend/src/features/settings.tsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  const handleEndGame = () => {
+    dispatch(actions.setIsGameLoading(true))
+    sendMessage({
+      action: FrontendWebsocketActions.GameEnded,
+      roomId,
+    })
+  }
+
+  return (
+    <motion.div initial={false} onClick={() => setIsExpanded(!isExpanded)}>
```
Comment (emilyhfdong) [inline frontend/src/features/settings.tsx:46]: :its_so_beautiful:

**Review by emilyhfdong — APPROVED** [review]

File: backend/src/functions/onCardDrawn.ts (hunk 1 above)
Comment (cindyhalim) [reply backend/src/functions/onCardDrawn.ts]: good point!

**Pull request merged**